<launch>
    <arg name="max_distance" default="1.5" />
    <arg name="min_distance" default="0.2" />

    <remap from="input" to="/camera/point_cloud"/>

    <node name="pcl_filter" pkg="pcl_filter" type="pcl_filter_node" output="screen">
        <param name="max_distance"   value="$(arg max_distance)" />
        <param name="min_distance"   value="$(arg min_distance)" />
    </node>
</launch>
